easybuild/easyconfigs/e/enable/enable-20160817_kiva.patch

```diff
@@ -0,0 +1,29 @@
+#non-git version (without .git directory) build fails
```

Comment:
@boegel Somewhere in the code it is buried to read version info from `.git` directory. 
I just made `kiva` use the version of `enable` (as far as I know, kiva is not distributed as standalone package, see next comment fo rthe next line). I took the inspiration from a 2 years old PR, which never been merged. 
If you use `git clone` instead of downloading the actual commit, you do not need the patch.